project/ScalaLibraryPlugin.scala

```diff
@@ -102,6 +104,25 @@ object ScalaLibraryPlugin extends AutoPlugin {
     } (Set(jar)), target)
   }
 
+  /* Remove Scala 2 Pickles from Classfiles */
+  private def unpickler(bytes: Array[Byte]): Array[Byte] = {
```

Comment:
This is a very confusing name, as the "unpickler" in the compiler is the thing that actually decodes things. Consider
```suggestion
  private def removeScala2Pickles(bytes: Array[Byte]): Array[Byte] = {
```